src/content/api/legacy-payment-requests.mdx

```diff
@@ -0,0 +1,806 @@
+      The asset type does not support refunds.
+    </Error>
+    <Error code="403" message="REFUND_WINDOW_EXCEEDED">
+      The time since the payment exceeds the window of time a transaction can be refunded in.
+    </Error>
+  </Properties>
+
+</Endpoint>
+
+---
+
+## Webhooks <Badge type="deprecated"/>
+
+Webhook notifications are sent for significant Payment life-cycle
+events. The Webhook endpoint is notified by sending an HTTP POST request to the
+`notifyUrl` defined in the Payment Request.
+
+### Life-cycle Events That Trigger Webhooks
+
+The supported event types that will be notified to the Payment Requests webhook
+and the associated "transactionType" value that will be sent in the payload
+are:
+
+
+| Event Type                | Value of "transactionType" |
+|:--------------------------|:---------------------------|
+| Payment Request Cancelled | CANCELLED                  |
+| Payment Request Expired   | EXPIRED                    |
+| Transaction Completed     | PURCHASE                   |
+| Transaction Refunded      | REFUND                     |
+
+
+#### Payment Request Cancelled
+
+A payment request can be cancelled by either calling the [requests.cancel](#cancel-a-payment-request)
+or [requests.void](#void-a-payment-request) endpoint before a request has been paid successfully.
+When a request has been cancelled we send a JWT that when decoded matches the
+*Payment Request Cancelled* example in the Decoded Webhook JWT Examples
+section below.
+
+#### Payment Request Expired
+
+A payment request expires two minutes after being created if it hasn't been
+cancelled, or paid. When a request has expired we send a JWT that when
+decoded matches the *Payment Request Cancelled* example in the Decoded
+Webhook JWT Examples section below with the `transactionType` set to EXPIRED.
+
+#### Transaction Completed
+
+A transaction is considered complete when [requests.pay](#pay-a-payment-request) is called with
+parameters that satisfy a payment request and the request has been paid
+successfully. When a transaction has been completed we send a JWT that when
+decoded matches the *Transaction Completed* example in the Decoded Webhook
+JWT Examples section below.
+
+#### Transaction Refunded
+
+A transaction can be refunded one to many times and each time a transaction
+has been refunded successfully we notify the webhook associated with the
+original payment request. A transaction can be refunded when
+[transactions.refund](#refund-a-transaction) has been called for a partial or full refund, or when
+[requests.void](#void-a-payment-request) is called for a request that has been paid. When a
+transaction has been refunded we send a JWT that when decoded matches the
+*Transaction Completed* example in the Decoded Webhook JWT Examples section
+below but with `transactionType` set to REFUND.
+
+### Webhook Payload
+
+The body of the webhook is a JSON document with the following format:
+
+<CodePanel>
```

Comment:
Suggestion: These could have titles

Reply:
@AndyClifford this defaults to `Example` which I think is okay, unless you had a more specific title in mind?